app/Http/Controllers/frontEndController.php

```diff
@@ -33,7 +32,6 @@ public function __construct(RepositorieFactory $repositorieFactory)
         $this->_inschrijvenRepository = $repositorieFactory->getRepositorie(RepositorieFactory::$INSCHRIJVENREPOKEY);
         $this->_agendaRepository = $repositorieFactory->getRepositorie(RepositorieFactory::$AGENDAITEMREPOKEY);
         $this->_userRepository = $repositorieFactory->getRepositorie(RepositorieFactory::$USERREPOKEY);
-        $this->_bookRepository = $repositorieFactory->getRepositorie(RepositorieFactory::$BOOKREPOKEY);
```

Comment:
bookrepository verwijderd, lijkt me niet de bedoeling

Reply:
fixed in commit b20ea05